### PR TITLE
Update docs 2

### PR DESCRIPTION
**Pull request by Lavanyagaur22**

### CODE REVIEW

File: docs/conflict_resolution.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ### Steps to support Conflict Resolution in your app:
 
-- Create a class, let say `UserConflictResolutionHandler` which extends ConfliceResolutionInterface provided by the library.
+- Create a class, let say `UserConflictResolutionHandler` which extends `ConfliceResolutionInterface` provided by the library.
 - Override the `resolveConflict()` method of the interface in your class.
```
Comment (wtrocki) [inline docs/conflict_resolution.md:6]: Implement instead of override

File: docs/conflict_resolution.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ### Steps to support Conflict Resolution in your app:
 
-- Create a class, let say `UserConflictResolutionHandler` which extends ConfliceResolutionInterface provided by the library.
```
Comment (wtrocki) [inline docs/conflict_resolution.md:5]: Make it a list

File: docs/conflict_resolution.md [hunk 3]
```diff
@@ -37,43 +37,18 @@ class UserConflictResolutionHandler(val context: Context) : ConflictResolutionIn
...
-                        }
-                    }
-                    mutationCall?.enqueue(callback)
-                }
-                
-                "UpdateUserMutation" -> {
-                    /* Similar for UpdateUserMutation
-                    1. Get all the fields from the clientState.
+                    /* 
+                    1. Get the necessary fields from the clientState.
```
Comment (wtrocki) [inline docs/conflict_resolution.md:48]: Kinda liked that we had some practical example. Maybe we can link to sample app for more practical one.

Comment (Lavanyagaur22) [reply docs/conflict_resolution.md:48]: Did you look at the earlier example? Was it okay? (the version based one?)

File: docs/conflict_resolution.md (hunk 2 above)
Comment (Lavanyagaur22) [reply docs/conflict_resolution.md:5]: Umm, List where?

File: docs/conflict_resolution.md (hunk 3 above)
Comment (wtrocki) [reply docs/conflict_resolution.md]: my bad. Yes it is duplicated

Comment (wtrocki) [reply docs/conflict_resolution.md:48]: my bad. Yes it is duplicated

**Review by harshithdwivedi — APPROVED** [review]

**Pull request merged**